.changeset/wet-beds-change.md

```diff
@@ -0,0 +1,7 @@
+---
+"myst-common": minor
+"myst-cli": minor
```

Comment:
I think this is the most notable part of this PR - making a breaking change for some naming consistency.